lib/appbundler/app.rb

```diff
@@ -116,14 +116,15 @@ def binstub(bin_file)
     end
 
     def load_statement_for(bin_file)
+      name, version = app_spec.name, app_spec.version
+      bin_basename = File.basename(bin_file)
       <<-E
-bin_dir = File.dirname(__FILE__)
-if File.symlink?(__FILE__)
-  bin_dir = File.dirname(File.readlink(__FILE__))
-end
+gem "#{name}", "= #{version}"
```

Comment:
I think the rest of the world is handled here:

https://github.com/chef/appbundler/blob/use-installed-gems/lib/appbundler/app.rb#L106